src/injector/inject.ts

```diff
@@ -139,35 +201,48 @@ export function inject (vm: ChibiCompatibleVM) {
 
     const originSetLocaleFunc = vm.setLocale;
     vm.setLocale = function (locale: string, ...args: unknown[]) {
+        lang = setup(locale);
         // @ts-expect-error internal hack
         const result = originSetLocaleFunc.call(this, locale, ...args);
         // @ts-expect-error lazy to extend VM interface
         vm.emit('LOCALE_CHANGED', locale);
         return result;
     };
-    
-    const originalArgReporterBooleanFunc = vm.runtime._primitives['argument_reporter_boolean'];
-    vm.runtime._primitives['argument_reporter_boolean'] = function (args: Record<string, unknown>, ...otherArgs: unknown[]) {
+    // TODO: compiler support
+    const originalArgReporterBooleanFunc =
+        vm.runtime._primitives['argument_reporter_boolean'];
+    vm.runtime._primitives['argument_reporter_boolean'] = function (
+        args: Record<string, unknown>,
+        ...otherArgs: unknown[]
+    ) {
         const chibiFlag = args.VALUE;
         switch (chibiFlag) {
-        case '🧐 Chibi Installed?':
-            return true;
-        default:
-            return originalArgReporterBooleanFunc.call(this, args, ...otherArgs);
+            case '🧐 Chibi?':
```

Comment:
I'm concerned that this change will break compatibility with previous versions...

Reply:
Yes it will. I will fix this later.

Reply:
Since the block cannot be translated, I used a universal name.